### PR TITLE
docs(error-boundary): fixing concept link from component

### PR DESCRIPTION
**Pull request by mateoguzmana**
## Describe your PR

Correcting a small link that points to a non-existent URL

### CODE REVIEW

File: boilerplate/app/screens/ErrorScreen/ErrorBoundary.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ interface State {
  * class component because according to the documentation, only class
  * components can be error boundaries.
  *
- * - [Documentation and Examples](https://github.com/infinitered/ignite/blob/master/docs/Error-Boundary.md)
+ * - [Documentation and Examples](https://github.com/infinitered/ignite/blob/master/docs/concept/Error-Boundary.md)
```
Comment (frankcalise) [inline boilerplate/app/screens/ErrorScreen/ErrorBoundary.tsx:20]: @mateoguzmana Thanks for the find! I think let's link directly to the hosted docs page now that it's up

https://docs.infinite.red/ignite-cli/concept/Error-Boundary/

**Review by frankcalise — CHANGES_REQUESTED** [review]

File: boilerplate/app/screens/ErrorScreen/ErrorBoundary.tsx (hunk 1 above)
Comment (mateoguzmana) [inline boilerplate/app/screens/ErrorScreen/ErrorBoundary.tsx:20]:
```suggestion
 * - [Documentation and Examples](https://docs.infinite.red/ignite-cli/concept/Error-Boundary/)
```

Comment (mateoguzmana) [reply boilerplate/app/screens/ErrorScreen/ErrorBoundary.tsx]: Nice, applied!

**Review by frankcalise — APPROVED** [review]

**Pull request merged**